news/_posts/core-weekly/2019-09-29-coreweekly-week-39-2019.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 39 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-10-01 16:50:00
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 23th of September to Sunday 29th of September 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+[...]
```

Comment:
This is the core weekly for last week, I think we can add some content. Maybe we can talk about the new core weekly generator ?

Reply:
yes, good idea !